src/operators/__tests__/operators.test.ts

```diff
@@ -53,6 +54,18 @@ describe('class.operators', () => {
                 bar: 'bar',
             });
         });
+        it('picks a subset of fields to validate', async () => {
+            const plain = {
+                bar: 'bar',
+                baz: 'baz',
+            };
+
+            const fooErrors = await validate(plainToClass(Foo, plain));
+            expect(fooErrors).toHaveLength(0);
+
+            const barErrors = await validate(plainToClass(Bar, plain));
```

Comment:
Prior to the changes in this PR, these second validators would fail because the the transformer layer (`plainToClass`) would omit fields that the parent class would validate as non-optional.